src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,97 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+interface Props {
+  peoples: Person[];
+  selectedPersonSlug: string;
+}
+
+export const PeopleTable: FC<Props> = ({
+  peoples,
+  selectedPersonSlug,
+}) => {
+  return (
+    <div className="block">
+      <div className="box table-container">
+        {peoples.length === 0 ? (
+          <p data-cy="noPeopleMessage">
+            There are no people on the server
+          </p>
+        ) : (
+          <table
+            data-cy="peopleTable"
+            className="table is-striped is-hoverable is-narrow is-fullwidth"
+          >
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>Sex</th>
+                <th>Born</th>
+                <th>Died</th>
+                <th>Mother</th>
+                <th>Father</th>
```

Comment:
Here you can cheat a little and make the code more readable and scalable if we take out the names of the columns in a separate array and map it as we do with `people`.